README.rst

```diff
@@ -148,12 +148,14 @@ You can run it with Docker as well
   # Download the image
   $ docker pull cloudcustodian/c7n
 
-  # Run the policy
+  # Run the policy (AWS)
   $ docker run -it \
-      -v $(pwd)/output:/output \
-      -v $(pwd)/policy.yml:/policy.yml \
-      --env-file <(env | grep "^AWS") \
-      cloudcustodian/c7n run -v -s /output /policy.yml
+    -v $(pwd)/output:/output \
```

Comment:
no leading slash on output, my next set of changes to the docker images will drop support for running as root, which means that won't work. the workdir for the images will be /home/custodian